input/fsh/bundlescompositions/CompositionProviderFetalDeathReportNew.fsh

```diff
@@ -54,7 +54,7 @@ Description: "This Composition profile contains constraints to address the use c
 * insert BFDRCompositionSectionEntrySlicing(fetus, 0, 76400-1, Provider Fetal Death)
 * insert BFDRCompositionSectionSlice(fetus, deliveryWeight, 0, 1, Delivery weight, The weight of the infant/fetus at birth/delivery, ObservationBirthWeight)
 * insert BFDRCompositionSectionSlice(fetus, gestationalAgeAtDelivery, 0, 1, Obstetric estimate of gestation, The obstetric estimate of the infant's gestation in completed weeks based on the birth/delivery attendant's final estimate of gestation which should be determined by all perinatal factors and assessments such as ultrasound\, but not the neonatal exam, ObservationGestationalAgeAtDelivery)
-* insert BFDRCompositionSectionSlice(fetus, methodOfDisposition, 0, 1, Method of decedent fetus disposition, Method of decedent fetus disposition, ObservationMethodOfDisposition) 
+* insert BFDRCompositionSectionSlice(fetus, ObservationFetalRemainsDispositionMethod, 0, 1, Method of decedent fetus disposition, Method of decedent fetus disposition, ObservationFetalRemainsDispositionMethod) 
```

Comment:
The first argument is the short name, not the name of the profile.
So, the first argument should be fetalRemainsDispositionMethod.